src/custom.d.ts

```diff
@@ -409,3 +409,24 @@ interface Transaction {
     gas: string
   }
 }
+
+interface Cryptocurrency {
```

Comment:
this has been renamed to `Chain` and already exist in extension, since I don't think `Cryptocurrency` is a good name for this. A chain can have multiple native currencies